modules/integration/common-utils/pom.xml

```diff
@@ -0,0 +1,229 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Fixed in https://github.com/wso2/product-app-manager/commit/f021f86c3c398dddeeca17e225a37e42f7f7033b